doc/content/getting-started/console/_index.md

```diff
@@ -5,3 +5,5 @@ aliases: [/guides/getting-started/cli]
 ---
 
 The Console is the management application of {{% tts %}} for LoRaWAN. It is a web application which can be used to register applications, end devices or gateways, monitor network traffic, or configure network related options, among other things.
+
+Visit the addresses page for [The Things Network]({{< ref "getting-started/ttn/addresses" >}}) or [{{% tts %}} Cloud]({{< ref "getting-started/ttn/addresses" >}}) to find the URL of the Console for these deployments.
```

Comment:
Idem

Reply:
Do we need a shortcode?

Reply:
I think we can indeed consider getting rid of the shortcode now that we no longer need to do the whole "The Things Stack" / "The Things Enterprise Stack" replacement magic anymore.

Reply:
> Do we need a shortcode?

Probably, but the cases are a bit different - find the URL of the console, find the CLI configuration.

Reply:
@johanstokking @htdvisser are you talking about the same thing? @johanstokking are you suggesting we use a shortcode for "Visit the addresses page for [The Things Network]({{< ref "getting-started/ttn/addresses" >}}) or [{{% tts %}} Cloud]({{< ref "getting-started/ttn/addresses" >}}) to find endpoints for these deployments" or suggesting that we remove the `tts` shortcode?

Reply:
Good question...

I don't think we need a shortcode for those couple of times we write this particular text.

Reply:
I think we can get rid of `{{% tts %}}` as it's not necessary anymore. We should just keep keeping an eye on things like "the stack", but take those extra seconds spelling the product name out.

My comment was indeed about the addresses. I see it three times here, and that in itself is not a reason for a shortcode, but I can imagine that referring to addresses will be of common use. Hence having that available in a shortcode could be useful.